applications/GeoMechanicsApplication/tests/cpp_tests/test_solving_strategy_factory.cpp

```diff
@@ -111,6 +111,7 @@ KRATOS_TEST_CASE_IN_SUITE(Create_ReturnsSolvingStrategy_ForNewtonRhapsonStrategy
     const auto created_strategy = SolvingStrategyFactoryType::Create(
             parameters, dummy_model_part);
 
+    KRATOS_EXPECT_TRUE(created_strategy->GetMoveMeshFlag()) // since it is set to true in the parameters
```

Comment:
Unfortunately, this is the only flag we can check

Reply:
Then this is the best you can do now :-)